lib/WorkflowMain.groovy

```diff
@@ -63,35 +63,35 @@ class WorkflowMain {
     private static void dadareftaxonomyExistsError(params, log) {
         if (params.dada_ref_databases && params.dada_ref_taxonomy && !params.dada_ref_databases.containsKey(params.dada_ref_taxonomy)) {
             def error_string = "=============================================================================\n" +
-                "  DADA2 reference database '${params.dada_ref_taxonomy}' not found in any config files provided to the pipeline.\n" +
+                "  DADA2 reference database '${params.dada_ref_taxonomy}' not found in any config file provided to the pipeline.\n" +
                 "  Currently, the available reference taxonomy keys for `--dada_ref_taxonomy` are:\n" +
                 "  ${params.dada_ref_databases.keySet().join(", ")}\n" +
                 "==================================================================================="
             Nextflow.error(error_string)
         }
     }
     //
-    // Exit pipeline if incorrect --qiime_ref_taxonomy key provided
+    // Exit pipeline if incorrect --sintax_ref_taxonomy key provided
     //
-    private static void qiimereftaxonomyExistsError(params, log) {
-        if (params.qiime_ref_databases && params.qiime_ref_taxonomy && !params.qiime_ref_databases.containsKey(params.qiime_ref_taxonomy)) {
+    private static void sintaxreftaxonomyExistsError(params, log) {
```

Comment:
No other reason than that I did those changes before summer, before someone else (you?) added the check to dev. Also, it felt logical to have sintax before qiime as that is the order we have them in the rest of the code, so I kept it like that. But I have no strong opinions here :)

Reply:
Ah I see, yes I added it as well recently, because I had the feeling its missing. I dont care about the sequence, makes sense to have the same sequence as elsewhere, fine with me.